README.md

```diff
@@ -128,4 +129,34 @@ You can then run the built binary with the example TOML file provided by setting
 
 ```console
 $ OLDAUDIT_CONFIG=./example.toml LD_AUDIT=./bazel-bin/src/main/libold.so ruby --help | head
-```
\ No newline at end of file
+```
+
+### Docker
+
+If you want to develop using a Docker container,  we provide a [Dockerfile](Dockerfile)
+with bazel ready to go! First, build the container:
+
+```bash
+$ docker build -t old .
+```
+
+You can then shell inside to use old:
+
+```bash
+$ docker run -it old
+```
+```bash
+$ apt-get install -y ruby
+$ LD_AUDIT=./bazel-bin/src/main/libold.so ruby -e "puts 'hi'"
+$ OLDAUDIT_CONFIG=./example.toml LD_AUDIT=./bazel-bin/src/main/libold.so ruby --help | head
+```
+
+or bind the present working directory (with the source code) to `/code`
```

Comment:
should this be bazel-code?

Reply:
No indeed not - the code is bound to `/code` in the container (a terrible convention I've had for as long as containers have been around!)